### PR TITLE
[BUILDING] contribution game v1.1.12

### PR DESCRIPTION
**Pull request by MaxHillebrand**
this ready for review [not yet ready for merge] branch adds the upcoming v1.1.12 review contribution game.

I also rearrange the page, so that all review games are under the same header.

### CODE REVIEW

File: docs/building-wasabi/ContributionGame.md [hunk 1]
```diff
@@ -42,21 +44,27 @@ This enabled many more reviewers to check out the latest additions to the softwa
...
 There were [detailed test vectors](https://github.com/zkSNACKs/WalletWasabi/issues/3396) to ensure that all the new additions were thoroughly covered in the review.
 This contribution game had a total bounty of up to `0.3 bitcoin`, sponsored by zkSNACKs.
 
 The total bounty ended up being distributed to 6 contributors, see the full results [here](https://github.com/zkSNACKs/WalletWasabi/issues/3396#issuecomment-610313893).
 
+### v1.1.12
+
+This version is already compatible with the v4 coordinator protocol.
+Thus, it is extraordinarily important that the release is thoroughly tested to ensure a smooth hardfork without major bugs.
+The [test vectors](https://github.com/zksnacks/walletwasabi/issues/3782) for the [release candidate](https://github.com/molnard/WalletWasabi/releases/tag/v1.1.12rc1) guide the testers what to focus on in their review.
```
Comment (yahiheb) [inline docs/building-wasabi/ContributionGame.md:59]: If I am not wrong the rc1 is just for internal testing, it is not related to the contribution game.

**Comment by yahiheb** [comment]
What about the `Education contribution game` was it cancelled? should we remove it?

File: docs/building-wasabi/ContributionGame.md (hunk 1 above)
Comment (MaxHillebrand) [reply docs/building-wasabi/ContributionGame.md:59]: Yes, I'll update it as soon as David pushes it.

**Comment by MaxHillebrand** [comment]
> What about the Education contribution game was it cancelled? should we remove it?

I don't know... will talk to the others and see, for sure out of scope of this PR, I'll follow up.

**Comment by MaxHillebrand** [comment]
- the final budget for the contribution game is `0.3 bitcoin`
- the release candidate used is David's RC`

In both cases, my initial commit in this branch is correct and good to merge.

The review game will start tomorrow on the 14th. 

Please give it one more round of review, then I merge and announce.

**Pull request merged**